### DISCUSSION TITLE
Does this work fine on Ubuntu 22.04?

### DISCUSSION THREAD

**Discussion by Mathiaszero** [AltraMayor/gatekeeper#650] [Q&A]
Hello, I was looking at the front page as well as releases and it doesn't seem that you support the latest Ubuntu version.  So will the program run fine with a build meant for the previous Ubuntu? Thanks in advance.

**Comment by AltraMayor** [comment]
The versions and recommendations found in the documentation are derived from production feedback. If you want to leverage the community's effort, you should follow those recommendations. Given the long cycle to develop and test Gatekeeper in production, every release of Gatekeeper will be behind the latest Ubuntu LTS version for the foreseeable future. This is not to say that Gatekeeper is written to only run under those conditions, just that those are the best-tested ones.

If you want to run Gatekeeper on a different version of the Ubuntu of the posted packages, I'd suggest building your packages for the version you want. Of course, this is a suggestion; you can try running the posted packages on a newer Ubuntu directly.
